util/__init__.py

```diff
@@ -54,10 +54,14 @@ def up(args, extra_args):
     if args.fake_data:
         env += ['export', 'INITIAL_DATA_SOURCE=FAKE_DATA', '&&']
         env += ['export', 'STREAM_DATA_SOURCE=NO_STREAMING', '&&']
+    if args.real_time:
```

Comment:
hmm. This name may be confusing as it's in case `fake_real_time`.

Reply:
Done